sql/clients_daily_v6_backfill_attribution.sql

```diff
@@ -0,0 +1,33 @@
+WITH
+  existing AS (
+  SELECT
+    * EXCEPT (attribution)
+  FROM
+    clients_daily_v6
+  WHERE
+    submission_date_s3 = @submission_date ),
+  main_pings AS (
+  SELECT
+    ROW_NUMBER() --
+    OVER (PARTITION BY client_id, submission_date_s3 ORDER BY `timestamp` ASC) --
```

Comment:
Are these `--` to manipulate auto-formatter behavior? Is so, I'm all for it and I think we'll get used to seeing these across the repo.

Reply:
Maybe we should document a "should" to run the BQ formatter and note that empty comments can be used to tweak formatting behavior.

Reply:
yes, but also this file wasn't supposed to be in this PR >_<